src/components/organisms/OcModal/OcModal.vue

```diff
@@ -157,6 +191,44 @@ export default {
         return value.match(/(outline|filled|raw)/)
       },
     },
+    /**
+     * Text of the secondary button
+     */
+    buttonSecondary: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
```

Comment:
I'm not sure whether I like this switch or not. 

My feeling is that it's a bit odd having to define `buttonSecondary` as `true` if I already provide `buttonSecondaryText`. Of course that makes it a bit easier to toggle, but maybe toggling the secondaryText is easy enough as well.